scripts/snapbuild.js

```diff
@@ -0,0 +1,30 @@
+const childProcess = require('child_process')
+const path = require('path')
+const rimraf = require('rimraf')
+
+const repoDir = path.dirname(__dirname)
+
+exports.buildBinary = () => {
+  childProcess.execFileSync('go', ['build', '-ldflags=-s -w', './cmd/snapshot'], { cwd: repoDir, stdio: 'ignore' })
+  return path.join(repoDir, process.platform === 'win32' ? 'snapshot.exe' : 'snapshot')
+}
+
+exports.removeRecursiveSync = path => {
+  try {
+    // Strangely node doesn't have a function to remove a directory tree.
```

Comment:
Node added [`fsPromises.rm`](https://nodejs.org/api/fs.html#fspromisesrmpath-options) in v14.4. Did you try this? What node version is this project leveraging?

Reply:
Oops. Copy and paste from how the `esbuild` side was doing things. Will fix this

Reply:
Updated to use built in `rm`: [`f972bb9` (#10)](https://github.com/cypress-io/esbuild/pull/10/commits/f972bb944f1be3e859325b53b7fee7b275ec5097)